src/Hero6.UserInterface.SierraVga/Entities/Padding.cs

```diff
@@ -0,0 +1,31 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="Padding.cs" company="LateStartStudio">
+//   Copyright (C) LateStartStudio
+//   This file is subject to the terms and conditions of the MIT license specified
+//   in the file 'LICENSE.CODE.md', which is a part of this source code package.
+// </copyright>
+// <summary>
+//   Defines the Padding struct.
+// </summary>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace LateStartStudio.Hero6.UserInterface.SierraVga.Entities
+{
+    public struct Padding
+    {
+        public float Left;
+        public float Top;
+        public float Right;
+        public float Bottom;
+                        
+        public Padding(float left, float top, float right, float bottom)
+        {
+            this.Left = left;
+            this.Top = top;
+            this.Right = right;
+            this.Bottom = bottom;
+        }
+
+        public Padding(float all) : this(all, all, all, all) { }
```

Comment:
I think you misunderstood my previous feedback, I'd like for it to so that the opening bracket is at its own line, and the closing bracket on its own line.

```
public Padding(float all) : this(all, all, all, all)
{
}
```

There are several reasons for this.

1. Of course StyleCop wants it.
2. I think it looks better, and also more consistent since this is how we do it with constructors that has content.
3. If we ever want to add content to this constructor it will create less "noise" on the git diff as it'll only show up as added lines between the brackets.